### ISSUE TITLE
java.lang.Exception

### ISSUE THREAD

**Issue by DarthTorus** [DraqueT/PolyGlot#1393]
![image](https://github.com/user-attachments/assets/a493f721-1b65-4730-9d07-adb6483379fa)

This happens every time I try to save. While I am ok with redoing the entirety of my language, I would prefer it actually saves. When I check, it hasn't. Which means something went wrong.

I also can't export to PDF....

**Comment by synthfi** [comment]
what OS/version of ployglot are you using?

**Comment by DarthTorus** [comment]
Whatever the latest is. On Dec 26, 2024, at 15:09, Kelly Schultz ***@***.***> wrote:﻿
what OS/version of ployglot are you using?

**Comment by TrapinchO** [comment]
I remember the error from a while back, notably in issues #1305 and #1195. I believe it was fixed, or at least there was an attempt to do so.

**Comment by DarthTorus** [comment]
> I remember the error from a while back, notably in issues #1305 and #1195. I believe it was fixed, or at least there was an attempt to do so.

Yeah I don't know what, if anything, I did wrong? I love the features but if I can't save anything, it's not worth using at the moment

**Comment by TrapinchO** [comment]
> > I remember the error from a while back, notably in issues #1305 and #1195. I believe it was fixed, or at least there was an attempt to do so.
> 
> Yeah I don't know what, if anything, I did wrong? I love the features but if I can't save anything, it's not worth using at the moment

Nope, I believe you did everything right.

**Comment by synthfi** [comment]
@DarthTorus if you're not on apple silicon, can you try with this release? https://github.com/DraqueT/PolyGlot/releases/tag/v3.6.1

**Comment by DarthTorus** [comment]
> @DarthTorus if you're not on apple silicon, can you try with this release? https://github.com/DraqueT/PolyGlot/releases/tag/v3.6.1

Ok so this works like a dream.... save for a very weird issue: When the orthography is auto generated, it doesn't switch from word to word. For example I made the word for "He". The orthography applies correctly. I switch to the word for say "to love", the conlang word is there, but the IPA is the pronunciation for "He"... like what?
